pyvisa-py/sessions.py

```diff
@@ -244,6 +244,27 @@ def after_parsing(self):
         """
         pass
 
+    def gpib_send_ifc(self):
+        """Pulse the interface clear line (IFC) for at least 100 microseconds.
+
+        Corresponds to viGpibSendIFC function of the VISA library.
+
+        :param session: Unique logical identifier to a session.
+        :return: return value of the library call.
+        :rtype: :class:`pyvisa.constants.StatusCode`
+        """
```

Comment:
As not all sessions implement this method, it may make sense to return the appropriate return code to signal it.